src/main.ts

```diff
@@ -13,6 +13,7 @@ import * as psql from "./db";
 import { Op } from "sequelize";
 import * as Sentry from "@sentry/node";
 import { Captcha } from "./typings/types";
+import * as moment from "moment";
```

Comment:
https://github.com/you-dont-need/You-Dont-Need-Momentjs

This library is huge, we can do date comp with vanillaJS